src/lib/ai/services/together.ts

```
@@ -143,31 +382,147 @@ export function createTogetherAIService(
+                      finishReason: finalFinishReason,
+                    }
+                    yield finalChunk
+                    return
+                  }
+                } catch (parseError) {
+                  appLogger.warn('Failed to parse streaming response line', {
+                    line: trimmedLine,
+                    error: parseError instanceof Error ? parseError.message : String(parseError),
+                  })
+                  continue
+                }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)